From 995d85a2c147bc5b4ed7f2d99a435d81ef6cd626 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Thu, 19 Oct 2017 14:01:02 -0500
Subject: [PATCH 7/7] kernel: Fix a Compile Warning

Remove the declaration of an unused variable that
results in a compile warning (unused variable).

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 kernel/futex.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/futex.c b/kernel/futex.c
index 789ff46..883132d 100644
--- a/kernel/futex.c
+++ b/kernel/futex.c
@@ -2819,7 +2819,6 @@ static int futex_wait_requeue_pi(u32 __user *uaddr, unsigned int flags,
 {
 	struct hrtimer_sleeper timeout, *to = NULL;
 	struct rt_mutex_waiter rt_waiter;
-	struct rt_mutex *pi_mutex = NULL;
 	struct futex_hash_bucket *hb, *hb2;
 	union futex_key key2 = FUTEX_KEY_INIT;
 	struct futex_q q = futex_q_init;
-- 
2.7.4

